docs/fundamentals/code-analysis/style-rules/ide0241.md

```diff
@@ -0,0 +1,83 @@
+---
+title: "IDE0241: Nullable directive is unnecessary"
+description: "Learn about code analysis rule IDE0241: Nullable directive is unnecessary"
+ms.date: 07/24/2023
+ms.topic: reference
+f1_keywords:
+- IDE0241
+helpviewer_keywords:
+- IDE0241
+dev_langs:
+- CSharp
+---
+# Nullable directive is unnecessary (IDE0241)
+
+| Property                 | Value                                                 |
+|--------------------------|-------------------------------------------------------|
+| **Rule ID**              | IDE0241                                               |
+| **Title**                | Nullable directive is unnecessary                     |
```

Comment:
Tagging @sharwell to review documentation for IDE0240 and IDE0241

Reply:
I'm going to merge this now so I can start on the next batch of rule without conflicts, but feel free to leave feedback here and I'll address it in a separate PR.